### PR TITLE
Basic structure

### PR DESCRIPTION
**Pull request by j-sv**
Creates the basic structure of the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
 # go-notification-client
+
+This is the client library for the Notifiation API.
```
Comment (Thompadude) [inline README.md]:
```suggestion
This is the client library for the Notification API.
```

File: example/go.mod [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module github.com/SKF/go-notification-client/example
+
+go 1.17
```
Comment (Thompadude) [inline example/go.mod:3]: Should we use the latest?

Comment (j-sv) [reply example/go.mod:3]: I figured many clients haven't bumped to 1.19 yet, so enforcing it may cause problems.

Comment (Thompadude) [reply example/go.mod:3]: Makes sense!

**Review by Thompadude — DISMISSED** [review]

**Review by Thompadude — APPROVED** [review]

**Pull request merged**